### PR TITLE
fix box request specs

### LINKED ISSUE #272: Test suite help!
We started this application during a hack weekend, and haven't been able to wrangle our test suite back to green yet. ANY help on this is VERY much appreciated!!! Even if you submit something to make ONE test pass, we'd be grateful.

### PR DESCRIPTION
**Pull request by catalindrg**
Resolves #272
Fixes `box_request` specs

### Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by maebeale** [comment]
Hello, @catalindrg and thank you for this PR! Would you merge the latest changes from develop and resubmit?

**Comment by catalindrg** [comment]
I updated my branch. Now it includes a small change, but I see that the variable that was used is not necessary and I think it adds some unnecessary code.

Small change - but I think it upgrades the flow of the spec

**Comment by maebeale** [comment]
Thank you, @catalindrg ! Yes, you're right about the variable. Would you add a second expect here that verifies the request's status is also :reviewed?

File: spec/models/box_request_spec.rb [hunk 1]
```diff
@@ -122,9 +122,8 @@
       box_request.reviewed_by_id = user.id
       box_request.claim_review
       box_request.reviewed_at = Time.now
-      @box = Box.create(box_request_id: box_request.id)
       box_request.complete_review
-      expect(box_request.box).to have_state(:reviewed)
+      expect(box_request.box).to have_state(:reviewed);
```
Comment (atzorvas) [inline spec/models/box_request_spec.rb:126]: we don't do `;` :)

**Review by atzorvas — APPROVED** [review]
please strip away `;` and this is good to go!

File: spec/models/box_request_spec.rb (hunk 1 above)
Comment (maebeale) [reply spec/models/box_request_spec.rb:126]: That will need to be done in a new PR. 
Will accept this one as is. 
Thanks so much for your help!

**Review by maebeale — APPROVED** [review]

**Pull request merged**